docs/reference/docker-image-support-matrix.rst

```diff
@@ -76,6 +92,57 @@ The following Docker image tags and associated inventories are validated for ROC
 
             .. tab-item:: Ubuntu 20.04
 
+               Tag
+                 `rocm/pytorch:rocm6.2.1_ubuntu20.04_py3.9_pytorch_release_2.3.0 <https://hub.docker.com/layers/rocm/pytorch/rocm6.2.1_ubuntu20.04_py3.9_pytorch_release_2.3.0/images/sha256-8ffc2fcde77c54800057b385bc95f1481fc5300dccff192afa865b29a5d00201>`_ (`latest <https://hub.docker.com/layers/rocm/pytorch/latest/images/sha256-63f46c1bd0db29c74238efa72778b4fabc614e180b8e9e3bcf8d457d9b34b610>`_)
```

Comment:
What's latest?  Can we add some text to explain how "latest" differs from "rocm/pytorch:rocm6.2.1_ubuntu20.04_py3.9_pytorch_release_2.3.0"?